src/crimson/os/seastore/transaction_manager.h

```diff
@@ -459,8 +459,6 @@ class TransactionManager : public ExtentCallbackInterface {
 	(intermediate_base == L_ADDR_NULL)
 	  == (intermediate_key == L_ADDR_NULL));
       if (ext) {
-        // FIXME: cannot and will not remap a dirty extent for now.
-        ceph_assert(!ext->is_dirty());
```

Comment:
@cyx1231st I came across an assert() error during a fio rbd test with delta_overwrite enabled. Do we need this assert()? I think dirty extent (OBJECT_DATA_BLOCK) can be dirty in delta_overwrite scenario.

Reply:
Introduced from https://github.com/ceph/ceph/pull/51179.

> I think dirty extent (OBJECT_DATA_BLOCK) can be dirty in delta_overwrite scenario.

Yeah, agreed.